packages/mobile/src/utils/formatting.ts

```diff
@@ -39,7 +39,8 @@ export const getExchangeRateDisplayValue = (value: BigNumber.Value): string => {
 
 export const getFeeDisplayValue = (value: BigNumber.Value | null | undefined): string => {
   return value
```

Comment:
Does value evaluate to false for any non-zero values? Wondering whether the case below is ever used for non-zero values

Reply:
When value is a `BigNumber` it would evaluate to true independently of the value in it.